docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -68,6 +68,7 @@ This file contains several service definitions:
   task instances once their dependencies are complete.
 - ``airflow-webserver`` - The webserver is available at ``http://localhost:8080``.
 - ``airflow-worker`` - The worker that executes the tasks given by the scheduler.
+- ``airflow-triggerer`` - The triggers task execution based on external events or conditions.
```

Comment:
Unfortunetly this is not correct description for `triggerer`, it is only for run [Deferrable Operators & Triggers](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/deferring.html#deferrable-operators-triggers)

Reply:
I understand that the `triggerer` process is essential for using deferrable operators, but it actually runs an `asyncio` event loop in your Airflow environment.

Reply:
My main point that this description could mislead end users, and they might decided that `triggerer` uses for some kind of Even Based Scheduling. But  in fact "external events" in is only deferred event from airflow worker.

Reply:
Very right. This is planned to happen in the future (as follow up after AIP-40) to be able to trigger execution of DAGs via external events, but it is not yet even agreed how to do it. Description like this could have mislead people that it's already possible today.

Reply:
Updated, thanks.